### ISSUE TITLE
DoneJS.com audit

### ISSUE THREAD

**Issue by chasenlehara** [donejs/donejs#728]
Go through the website and look for broken demos & links, missing images & videos, spelling & grammar issues, accessibility issues, poor performance, etc.

**Comment by matthewp** [comment]
👍 On accessibility, this is something we probably haven't put enough time into.

**Comment by 0joshuaolson1** [comment]
For testing accessibility, [plenty of tools](https://github.com/collections/web-accessibility) are a good starting point to fix that.

They would, however, miss more manual/design QA problems like the website's approach to feedback. ("This unlabelled logo looked like a contrast toggle, so I accidentally visited another website; also there's no direct link to an issue tracker.")

This seems like a PR issue to me, since I personally judge libraries/frameworks by the areas they remember like accessibility features/components/links/education.
